### PR TITLE
FAI-280 - fix dmn model model publisher

### PR DESCRIPTION
**Pull request by r00ta**
JIRA: https://issues.redhat.com/browse/FAI-280

The aim of this PR is to fix the bug https://issues.redhat.com/browse/FAI-280 with a workaround as suggested by the quarkus team https://github.com/quarkusio/quarkus/issues/12820 .

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by r00ta** [comment]
Related to https://github.com/kiegroup/kogito-examples/pull/430

File: addons/tracing/tracing-decision-quarkus-addon/src/main/java/org/kie/kogito/tracing/decision/QuarkusStartupModelPublisher.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@Singleton
+@Startup
+public class QuarkusStartupModelPublisher {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QuarkusStartupModelPublisher.class);
+
+    @Inject
+    QuarkusModelEventEmitter modelEventEmitter;
+
+    public void publish(@Observes StartupEvent event) {
```
Comment (danielezonca) [inline addons/tracing/tracing-decision-quarkus-addon/src/main/java/org/kie/kogito/tracing/decision/QuarkusStartupModelPublisher.java:37]: Can you add a comment to refer Quarkus issue?

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

File: addons/tracing/tracing-decision-quarkus-addon/src/main/java/org/kie/kogito/tracing/decision/QuarkusStartupModelPublisher.java (hunk 1 above)
Comment (r00ta) [reply addons/tracing/tracing-decision-quarkus-addon/src/main/java/org/kie/kogito/tracing/decision/QuarkusStartupModelPublisher.java:38]: Sure :+1:

**Review by manstis — APPROVED** [review]
Thank-you @r00ta

**Pull request merged**